go.mod

```diff
@@ -0,0 +1,72 @@
+module github.com/eclipse/che-operator
```

Comment:
Any idea why this change has been introduced ?

Reply:
Automatically generated is VS Code with go plugin and go 1.12 version